python/ray/dataframe/dataframe.py

```diff
@@ -33,10 +33,12 @@
     _blocks_to_col,
     _blocks_to_row,
     _build_columns,
-    _create_block_partitions)
+    _create_block_partitions,
+    _inherit_docstrings)
 from . import get_npartitions
 
 
+@_inherit_docstrings(pd.DataFrame)
```

Comment:
This can be used on other Dataframe related classes (e.g. DataFrameGroupBy)

Reply:
Added decorator `Series` and `DataFrameGroupBy`. We should make a note somewhere to override documentation for future Pandas classes that we implement.